man.Rmd

```diff
@@ -147,27 +190,49 @@ str_unique <- function(string, ...) {
 Here the introduction includes the title ("Remove duplicated strings") and a basic description of what the function does.
 The introduction is followed by five tags: two `@params`, one `@returns`, one `@seealso`, one `@examples`, and one `@export`.
 
-Note that each line of the block is 80 characters wide (matching the line length of the surrounding R code) and the second and subsequent lines of the long `@param` tag are indented, which makes the entire block easier to scan.
+Note that the block has an intentional line length (typically the same as that used for the surrounding R code) and the second and subsequent lines of the long `@param` tag are indented, which makes the entire block easier to scan.
```

Comment:
I don't want to promise a super-specific line length; better to express a general principle. The stringr examples aren't consistently line-wrapped at this point and we might actually need to fiddle with line length here in the book, because we're getting horizontal scrolling.